plugin/validator_test.go

```diff
@@ -2378,3 +2378,63 @@ func getErrForField(e error, path string) string {
 	}
 	return ""
 }
+
+func TestValidator_Transformations(t *testing.T) {
```

Comment:
What about the other properties, like `needs`? https://github.com/Kong/kong/blob/907a4a074a8da826eae7a83d4ce0c6698de07926/kong/db/schema/metaschema.lua#L141

Reply:
Not needed right now since`needs` is not used in any plugin schema. We can definitely add it when needed.